clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleHttpLoadBalancerAtomicOperation.groovy

```diff
@@ -105,7 +105,7 @@ class DeleteGoogleHttpLoadBalancerAtomicOperation extends DeleteGoogleLoadBalanc
     List<String> listenersToDelete = []
     projectForwardingRules.each { ForwardingRule rule ->
       def proxy = GCEUtil.getTargetProxyFromRule(compute, project, rule)
-      if (GCEUtil.getLocalName(proxy?.getUrlMap()) == urlMapName) {
+      if (proxy?.urlMap && GCEUtil.getLocalName(proxy?.urlMap) == urlMapName) {
```

Comment:
Doesn't `GCEUtil.getLocalName()` return `null` if the passed arg is empty? Not sure why this is necessary.